Sources/StreamVideoSwiftUI/ViewFactory.swift

```diff
@@ -49,7 +49,6 @@ public protocol ViewFactory: AnyObject {
     func makeVideoParticipantsView(
         viewModel: CallViewModel,
         availableSize: CGSize,
-        onViewRendering: @escaping (VideoRenderer, CallParticipant) -> Void,
```

Comment:
If we decide to change the public api, we should also update the docs and tutorials where this is used. But let's see if we can avoid that.